.github/workflows/tests-l4lb.yaml

```diff
@@ -0,0 +1,69 @@
+name: Cilium L4LB XDP
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  issue_comment:
+    types:
+      - created
+  # Run every 6 hours
+  schedule:
+    - cron:  '0 4/6 * * *'
+  ### FOR TESTING PURPOSES
+  # pull_request:
+  #  types:
+  #    - "labeled"
+  ###
+
+concurrency:
+  # In case of PR comment, we can't simply append the comment to the group name
+  # as that is too long and results in an error. Instead we check if it's a
+  # trigger phrase and append a simple 'trigger-phrase'.
+  group: |
+    "${{ github.workflow }} ${{ github.event.issue.pull_request.url || 'scheduled' }}
+     ${{ (startsWith(github.event.comment.body, 'ci-l4lb') ||
+          startsWith(github.event.comment.body, 'test-me-please')) && 'trigger-phrase' }}"
+  cancel-in-progress: true
+
+jobs:
+  setup-and-test:
```

Comment:
Job is missing an `if` condition, so by default it will run on every comment create as per the events defined at the top. If the intent is to use a specific trigger phrase (I think it is, considering the concurrency group above :D), we should guard with something like this:

```
    if: |
      (github.event.issue.pull_request && (
        startsWith(github.event.comment.body, 'ci-l4lb') ||
        (startsWith(github.event.comment.body, 'test-me-please'))
      )) ||
      (github.event_name == 'schedule' && github.repository == 'cilium/cilium') ||
      github.event.label.name == 'ci-run/l4lb'
```

Reply:
@brb BTW, and I recommend we do so, if you do add a label filter `ci-run/l4lb` as proposed in the code snippet above, you may test the workflow prior to merging by uncommenting the following lines in a separate commit/testing PR:

```
  # pull_request:
  #  types:
  #    - "labeled"
```

and then adding the `ci-run/l4lb` to the testing PR. This will trigger the `labeled` event as per the filter, and we can test that the workflow behaves as intended (mostly... since it's a different event, we can't test everything) before merging for real :)

Reply:
Thanks!